ui/core/components/suggest_reforges_action.tsx

```diff
@@ -507,8 +507,9 @@ export class ReforgeOptimizer {
 				}
 
 				const forcedProcInput = new EnumPicker(null, this.player, {
+					extraCssClasses: ["mb-2"],
 					id: 'reforge-optimizer-force-stat-proc',
-					label: 'Force Matrix/Apparatus proc',
+					label: 'RoRo proc',
```

Comment:
One additional change that is required for RoRo reforging compared to the prior Matrix/Apparatus implementation is to (a) set the EP of the targeted highest stat to 0, and (b) set the EPs of the two other stats to be equal to their mean, just before running the Reforger. This will force the algorithm to balance the stats to be as equal as possible in order to maximize the buff strength.